pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -467,6 +468,26 @@ func (c *Cluster) EnableServiceMonitor(activeDaemon string) error {
 	}
 	serviceMonitor.Spec.NamespaceSelector.MatchNames = []string{c.clusterInfo.Namespace}
 	serviceMonitor.Spec.Selector.MatchLabels = c.selectorLabels(activeDaemon)
+
+	// If the cephcluster is managed by another resource,
+	// add the name of that resource as a label on the ceph metrics.
+	if c.spec.Labels != nil {
+		if monitoringLabels, ok := c.spec.Labels["monitoring"]; ok {
+			if managedBy, ok := monitoringLabels["rook.io/managedBy"]; ok {
+				relabelConfig := monitoringv1.RelabelConfig{
+					TargetLabel: "managedBy",
+					Replacement: managedBy,
+				}
+				serviceMonitor.Spec.Endpoints[0].RelabelConfigs = append(
```

Comment:
won't `RelabelConfigs` need a `nil` check?

Reply:
@sp98 Can you please elaborate on that?

Reply:
appending to a nil slice would actually initialize the slice, so it looks good